app/views/rooms/_menu.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="card mt-4 w-10" style="height: 10rem">
+        <ul class="list-group list-group-flush">
+            <li class="list-group-item">
+                <%= link_to "Listings", listing_room_path %>
+                <% if @room.home_type? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Price", price_room_path %>
+                <% if @room.price? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Description", description_room_path %>
+                <% if @room.listing_name? || @room.summery? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Photos", photos_room_path %>
+                <% if @room.images.any? %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
+            <li class="list-group-item">
+                <%= link_to "Amenities", amenities_room_path %>
+                <% if @room.tv == true || @room.kitchen == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% elsif @room.internet == true || @room.heating == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% elsif @room.air_conditioning == true %>
+                    <%= raw("<i class='fas fa-check text-warning'></i>") %>
+                <% end %>
+            </li>
```

Comment:
you may add a check icon by default for Amenities since the requirements for the checking of publish does not expect a single amenity being checked